ArjunaCore/arjuna/classes/com/arjuna/ats/internal/arjuna/objectstore/hornetq/HornetqJournalEnvironmentBean.java

```diff
@@ -40,6 +40,8 @@
 
     private volatile int minFiles = 4;
 
+    private volatile int poolSize = -1;
```

Comment:
I just wonder How the poolSize uses in the artemis journal store and why it set to -1 ?

Thanks,
Amos